crates/resolver/src/resolver.rs

```diff
@@ -74,6 +78,18 @@ where
         self
     }
 
+    /// Set the DNSSEC trust anchors to be used by the resolver.
```

Comment:
This doc comment is copied from elsewhere.
```suggestion
    /// Set maximum limits on NSEC3 additional iterations.
    ///
    /// See [RFC 9276](https://www.rfc-editor.org/rfc/rfc9276.html). Signed
    /// zones that exceed the soft limit will be treated as insecure, and signed
    /// zones that exceed the hard limit will be treated as bogus.
```